deps/fmt.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "fmt",
+    "buildsystem": "cmake-ninja",
+    "builddir": true,
+    "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release"
+    ],
+    "cleanup": [
+        "/lib64"
```

Comment:
Seems odd?

Reply:
No it's a static library. It isn't required after the build.

Reply:
`cleanup: ["*"]` is more clear that nothing from a module is needed.

Reply:
I can remove it